setup.cfg

```
@@ -32,6 +32,7 @@ install_requires =
     pyerfa>=2.0
     PyYAML>=3.13
     packaging>=19.0
+	numba>=0.55.2
```

Indentation aside, this is quite a controversial proposal. Have you seen #12448  ? 😆 

```suggestion
    numba>=0.55.2
```